Source/Scene/Label.js

```diff
@@ -825,38 +957,58 @@ define([
      * @private
      */
     Label.getScreenSpaceBoundingBox = function(label, screenSpacePosition, result) {
```

Comment:
I haven't looked at the tests yet, but do we have enough tests to cover the new cases here?